dataminer-overview/Pricing/MACC_and_DataMiner_credits.md

```diff
@@ -2,21 +2,19 @@
 uid: MACC_and_DataMiner_credits
 ---
 
-# Maximizing your azure investments with MACC and DataMiner credits
+# Maximizing your azure investments with MACC and DataMiner Credits
```

Comment:
At present, credits is never capitalized in "DataMiner credits"; neither in our documentation, nor on the [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/skylinecommunicationsnv1644148291022.dataminer?tab=Overview). If we start capitalizing this now, we will be introducing an inconsistency, or else we'd have to adjust it everywhere, which will be quite a bit of work, so I'm not sure if this is a good idea.